src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -263,6 +241,38 @@ await listener.AfterExecutionAsync(context)
             return result;
         }
 
+        /// <summary>
+        /// Builds a <see cref="ExecutionContext"/> instance from the provided values.
+        /// </summary>
+        protected virtual ExecutionContext BuildExecutionContext(ExecutionOptions options, Document document, Operation operation, Variables variables, Metrics metrics)
+        {
+            var context = new ExecutionContext
+            {
+                Document = document,
+                Schema = options.Schema,
+                RootValue = options.Root,
+                UserContext = options.UserContext,
+
+                Operation = operation,
+                Variables = variables,
+                Fragments = document.Fragments,
```

Comment:
Let's remove `Fragments` property from `ExecutionContext` and `IResolveFieldContext`.  For some reason I missed it last time.

Reply:
I'll separately do a pair of PRs for this, after this is merged.

Reply:
OK. https://github.com/graphql-dotnet/graphql-dotnet/issues/2327